README.md

```diff
@@ -25,6 +25,10 @@ The official [AKS Engine documentation][docs] covers everything you need to know
 
 Want to get involved? The [community guide][community] covers everything you need to know about the AKS Engine community and how you can contribute.
 
+## Official Microsoft Azure Support
```

Comment:
maybe the title shouldn't be "official support" if the paragraph is saying there is no official azure support?

Reply:
just to avoid confusion for people that read fast and just look at titles